megamek/src/megamek/server/Server.java

```diff
@@ -3766,7 +3745,7 @@ public boolean accept(Entity entity) {
                 // If there's only one team moving, we don't need to bother
                 // with the evenTracker, just make sure the even turns are
                 // evenly distributed
-                numEven += (int) Math.round((teamEvenTurns / min) + 0.5);
+                numEven += (int) Math.ceil(teamEvenTurns / min);
```

Comment:
What I was trying to say is that what was originally here (`teamEvenTurns / min + 0.5`) is the equivalent of `(int) Math.round(teamEvenTurns / min)`. Changing it to `(int) Math.round((teamEvenTurns / min) + 0.5)` is the equivalent of `(int) Math.ceil(teamEvenTurns / min)`, which changes the logic. Unless the original logic is wrong, this should be 
```suggestion
                numEven += (int) Math.round(teamEvenTurns / min);
```

Reply:
Nope, I just did a late night change that doesn't make sense to me either. Fixing in the next few min.